test/models/test_video_vqvae.py

```diff
@@ -105,6 +124,17 @@ def test_forward(self, input_tensor, encoder):
         )
         assert_expected(actual, expected, rtol=0, atol=1e-4)
 
+    def test_get_latent_shape(self, downsampler, big_input):
```

Comment:
We could also combine `test_get_latent_shape` and `test_encoder_latent_shape` into one test with `actual` being the output of the forward pass and `expected` being that of `get_latent_shape`? Because ultimately we are more interested in the equality of the two.